genesis-simple-menus.php

```diff
@@ -36,7 +36,6 @@ function genesis_simple_menus() {
 	}
 
 	return $object;
-
```

Comment:
Because I had to [bump wpcs to 3](https://github.com/studiopress/genesis-simple-menus/pull/26/files#r1381120715), version 3 had errors for empty lines like these.